systemd/systemd.go

```diff
@@ -118,7 +118,7 @@ const (
 )
 
 var restartMap = map[string]RestartCondition{
-	"never":       RestartNever,
+	"no":          RestartNo,
```

Comment:
ah, I see why it is `never`, `no` gets translated to `false` in yaml without quotes or any other string indicator. Is that why this was done this way? It still is an invalid restart condition that would land on the template though